aspnet/migration/aspnetcoremigration.rst

```diff
@@ -0,0 +1,190 @@
+
+For information regarding a complete list of breaking changes in RC2 for ASP.NET, see the `ASP.NET Announcements page <https://github.com/aspnet/announcements/issues?q=is%3Aopen+is%3Aissue+milestone%3A1.0.0-rc2>`_
+
+Implement main with WebHostBuilder logic
+----------------------------------------
+
+Since ASP.NET core apps are just console apps, you must explicitly call ``Main`` to build up the host and tell it to start listening.
+
+.. code-block:: c#
+
+  public class Program
+  {
+    public static void Main(string[] args) 
+      => WebApplication.Run<Startup>(args);
+  }
+
+Configure the webroot and port in the ``hosting.json`` file.
+webroot definition moved from project.json to hosting.json
+
+WebHostBuilder API updates
+--------------------------
+
+All classes prefixed with WebApplication have been renamed to WebHost. This includes:
+
+===========================    =========================
+IWebApplicationBuilder         IWebHostBuilder
+WebApplicationBuilder          WebHostBuilder
+IWebApplication                IWebHost
+WebApplication                 WebHost
+WebApplicationOptions          WebHostOptions
+WebApplicationDefaults         WebHostDefaults
+WebApplicationService          WebHostService
+WebApplicationConfiguration    WebHostConfiguration
+===========================    =========================
+
+In the ``project.json`` file, ``web`` no longer exists as a command in the ``commands`` section. Use ``run`` or ``dotnet <dllname>`` instead  
+
+.. code-block:: JavaScript
+
+  "commands": {
+    "web": "Microsoft.AspNet.Server.Kestrel",
+    "ef": "EntityFramework.Commands"
+  },
+
+IHostingEnvironment changes 
+---------------------------
+
+All environment variables are now prefixed with the ``ASPNETCORE_`` prefix.
+
+======================  =========================    
+Old prefix              New prefix                           
+======================  =========================  
+ASPNET_WEBROOT          ASPNETCORE_WEBROOT
+ASPNET_SERVER           ASPNETCORE_SERVER
+ASPNET_APP              ASPNETCORE_APP
+ASPNET_ENVIRONMENT      ASPNETCORE_ENVIRONMENT
+ASPNET_DETAILEDERRORS   ASPNETCORE_DETAILEDERRORS
+======================  =========================  
+
+The hosting configuration keys are now consistent with the command line, environment variables, and ``hosting.json`` values. The ``Microsoft.AspNet.Hosting.json`` configuration file was renamed to ``hosting.json``.
+
+Namespace and package name changes
+---------------------------------- 
+
+All Microsoft.AspNet.* namespaces are renamed to Microsoft.AspNetCore.*. 
+The EntityFramework.* packages and namespaces are changing to Microsoft.EntityFrameworkCore.*.
+The version numbers of the above are being reset to 1.0.0-*.
+
+Working with IIS
+----------------
+
+Middleware is now setup using ``WebHostBuilder`` and no longer called in the ``Configure`` method of the ``Startup`` class.
+
+Breaking changes in the ``publish-iis`` tool: 
```

Comment:
@danroth27  should errors in the original announcement that use the same verbiage as here also be changed? Or some kind of note to clarify since you want links to the actual announcements? I  can see where it would be confusing, for example, the announcement says it is a "breaking change".  https://github.com/aspnet/Announcements/issues/169